src/test/java/openfasttrack/cli/TestCommandLineInterpreter.java

```diff
@@ -147,6 +168,26 @@ public void testCombinedParamters()
         assertThat(stub.getC(), equalTo(StubEnum.VALUE2));
     }
 
+    @Test
+    public void testChainedSingleCharacterParameters()
+    {
+        final CommandLineArgumentsStub stub = parseArguments("-bda", "value_a", "value_1",
+                "value_2");
+        assertThat(stub.getUnnamedValues(), equalTo(asList("value_1", "value_2")));
+        assertThat(stub.isB(), equalTo(true));
+        assertThat(stub.isD(), equalTo(true));
+        assertThat(stub.getA(), equalTo("value_a"));
+    }
+
+    @Test
+    public void testChainedSingleCharacterParametersMustFailIfNonBooleanMisplaced()
+    {
+        final CommandLineArgumentsStub stub = new CommandLineArgumentsStub();
+        expectParseException(stub, asList("-bad", "value_a"), "No value for argument 'a'");
+        assertThat(stub.isB(), equalTo(true));
+        assertThat(stub.isD(), equalTo(true));
```

Comment:
Removed since it does not matter whether the remaining parameters are detected correctly when an exception is thrown.